### PR TITLE
[OpenXLA]Delete profiler_pywrap_impl from TensorFlow

### PR DESCRIPTION
**Pull request by ManfeiBai**
The only `"@org_tensorflow//tensorflow/python/profiler/internal:profiler_pywrap_impl"`'s use is `tensorflow::profiler::pywrap::Trace` in `torch_xla/csrc/init_python_bindings.cpp`, so delete this useage for deprecate Tensorflow;

checked locally in TPUVM after modify:

```
export PJRT_DEVICE=TPU

cd pytorch/

python setup.py install --user

cd xla/

BUILD_CPP_TESTS=0 TPUVM_MODE=1 python setup.py develop --user

pip install torch-xla[tpuvm] --user

python3 test/test_profile_mp_mnist.py --fake_data --num_epochs 1
```

### CODE REVIEW

**Comment by ManfeiBai** [comment]
local test result of `python3 test/test_profile_mp_mnist.py --fake_data --num_epochs 1`:
```
(pytorch) root@56595ce5bd16:/pytorch/xla# python3 test/test_profile_mp_mnist.py --fake_data --num_epochs 1
Epoch 1 train begin 05:24:40
| Training Device=xla:1/1 Step=0 Loss=nan Rate=304.89 GlobalRate=304.88 Time=05:24:41
| Training Device=xla:1/5 Step=0 Loss=nan Rate=319.73 GlobalRate=319.72 Time=05:24:41
| Training Device=xla:1/7 Step=0 Loss=nan Rate=243.32 GlobalRate=243.31 Time=05:24:41
| Training Device=xla:1/3 Step=0 Loss=nan Rate=306.71 GlobalRate=306.70 Time=05:24:41
| Training Device=xla:0/0 Step=0 Loss=nan Rate=337.83 GlobalRate=337.82 Time=05:24:41
| Training Device=xla:0/6 Step=0 Loss=nan Rate=240.19 GlobalRate=240.19 Time=05:24:41
| Training Device=xla:0/4 Step=0 Loss=nan Rate=282.63 GlobalRate=282.63 Time=05:24:41
| Training Device=xla:0/2 Step=0 Loss=nan Rate=299.71 GlobalRate=299.70 Time=05:24:41
| Training Device=xla:1/7 Step=20 Loss=0.10472 Rate=129.45 GlobalRate=55.60 Time=05:25:28
| Training Device=xla:1/1 Step=20 Loss=0.10472 Rate=154.08 GlobalRate=55.72 Time=05:25:28
| Training Device=xla:0/2 Step=20 Loss=0.09676 Rate=152.00 GlobalRate=55.72 Time=05:25:28
| Training Device=xla:1/3 Step=20 Loss=0.10472 Rate=154.80 GlobalRate=55.73 Time=05:25:28
| Training Device=xla:0/0 Step=20 Loss=0.09676 Rate=167.25 GlobalRate=55.77 Time=05:25:28
| Training Device=xla:0/6 Step=20 Loss=0.09676 Rate=128.20 GlobalRate=55.59 Time=05:25:28
| Training Device=xla:1/5 Step=20 Loss=0.10472 Rate=160.01 GlobalRate=55.74 Time=05:25:28
| Training Device=xla:0/4 Step=20 Loss=0.09676 Rate=145.17 GlobalRate=55.68 Time=05:25:28
| Training Device=xla:1/1 Step=40 Loss=0.00549 Rate=92.65 GlobalRate=53.69 Time=05:26:18
| Training Device=xla:1/3 Step=40 Loss=0.00549 Rate=92.94 GlobalRate=53.69 Time=05:26:18
| Training Device=xla:0/6 Step=40 Loss=0.00483 Rate=82.30 GlobalRate=53.63 Time=05:26:18
| Training Device=xla:0/0 Step=40 Loss=0.00483 Rate=97.92 GlobalRate=53.71 Time=05:26:18
| Training Device=xla:0/2 Step=40 Loss=0.00483 Rate=91.82 GlobalRate=53.68 Time=05:26:18
| Training Device=xla:1/7 Step=40 Loss=0.00549 Rate=82.80 GlobalRate=53.63 Time=05:26:18
| Training Device=xla:1/5 Step=40 Loss=0.00549 Rate=95.03 GlobalRate=53.70 Time=05:26:18
| Training Device=xla:0/4 Step=40 Loss=0.00483 Rate=89.09 GlobalRate=53.67 Time=05:26:18
| Training Device=xla:0/2 Step=60 Loss=0.00218 Rate=65.16 GlobalRate=51.44 Time=05:27:12
| Training Device=xla:0/0 Step=60 Loss=0.00218 Rate=67.60 GlobalRate=51.46 Time=05:27:12
| Training Device=xla:1/7 Step=60 Loss=0.00245 Rate=61.55 GlobalRate=51.41 Time=05:27:12
| Training Device=xla:1/1 Step=60 Loss=0.00245 Rate=65.49 GlobalRate=51.45 Time=05:27:12
| Training Device=xla:1/3 Step=60 Loss=0.00245 Rate=65.61 GlobalRate=51.45 Time=05:27:12
| Training Device=xla:0/6 Step=60 Loss=0.00218 Rate=61.35 GlobalRate=51.41 Time=05:27:12
| Training Device=xla:1/5 Step=60 Loss=0.00245 Rate=66.44 GlobalRate=51.45 Time=05:27:12
| Training Device=xla:0/4 Step=60 Loss=0.00218 Rate=64.07 GlobalRate=51.43 Time=05:27:12
| Training Device=xla:0/2 Step=80 Loss=0.00134 Rate=58.55 GlobalRate=52.08 Time=05:27:59
| Training Device=xla:1/1 Step=80 Loss=0.00155 Rate=58.68 GlobalRate=52.08 Time=05:27:59
| Training Device=xla:1/7 Step=80 Loss=0.00155 Rate=57.10 GlobalRate=52.06 Time=05:27:59
| Training Device=xla:1/3 Step=80 Loss=0.00155 Rate=58.73 GlobalRate=52.09 Time=05:27:59
| Training Device=xla:0/0 Step=80 Loss=0.00134 Rate=59.52 GlobalRate=52.10 Time=05:27:59
| Training Device=xla:0/6 Step=80 Loss=0.00134 Rate=57.02 GlobalRate=52.05 Time=05:27:59
| Training Device=xla:1/5 Step=80 Loss=0.00155 Rate=59.06 GlobalRate=52.09 Time=05:27:59
| Training Device=xla:0/4 Step=80 Loss=0.00134 Rate=58.11 GlobalRate=52.07 Time=05:27:59
| Training Device=xla:0/0 Step=100 Loss=0.00092 Rate=56.25 GlobalRate=52.47 Time=05:28:47
| Training Device=xla:1/3 Step=100 Loss=0.00107 Rate=55.93 GlobalRate=52.47 Time=05:28:47
| Training Device=xla:1/7 Step=100 Loss=0.00107 Rate=55.28 GlobalRate=52.44 Time=05:28:47
| Training Device=xla:0/2 Step=100 Loss=0.00092 Rate=55.86 GlobalRate=52.46 Time=05:28:47
| Training Device=xla:0/6 Step=100 Loss=0.00092 Rate=55.25 GlobalRate=52.44 Time=05:28:47
| Training Device=xla:1/1 Step=100 Loss=0.00107 Rate=55.91 GlobalRate=52.47 Time=05:28:47
| Training Device=xla:0/4 Step=100 Loss=0.00092 Rate=55.68 GlobalRate=52.46 Time=05:28:47
| Training Device=xla:1/5 Step=100 Loss=0.00107 Rate=56.06 GlobalRate=52.47 Time=05:28:47
| Training Device=xla:1/1 Step=120 Loss=0.00083 Rate=55.04 GlobalRate=52.79 Time=05:29:34
| Training Device=xla:1/7 Step=120 Loss=0.00083 Rate=54.79 GlobalRate=52.77 Time=05:29:34
| Training Device=xla:0/2 Step=120 Loss=0.00071 Rate=55.02 GlobalRate=52.78 Time=05:29:34
| Training Device=xla:0/0 Step=120 Loss=0.00071 Rate=55.18 GlobalRate=52.79 Time=05:29:34
| Training Device=xla:1/3 Step=120 Loss=0.00083 Rate=55.05 GlobalRate=52.79 Time=05:29:34
| Training Device=xla:0/6 Step=120 Loss=0.00071 Rate=54.78 GlobalRate=52.76 Time=05:29:34
| Training Device=xla:0/4 Step=120 Loss=0.00071 Rate=54.95 GlobalRate=52.78 Time=05:29:34
| Training Device=xla:1/5 Step=120 Loss=0.00083 Rate=55.10 GlobalRate=52.79 Time=05:29:34
| Training Device=xla:0/2 Step=140 Loss=0.00057 Rate=115.43 GlobalRate=58.24 Time=05:29:50
| Training Device=xla:0/6 Step=140 Loss=0.00057 Rate=115.33 GlobalRate=58.22 Time=05:29:50
| Training Device=xla:0/0 Step=140 Loss=0.00057 Rate=115.49 GlobalRate=58.25 Time=05:29:50
| Training Device=xla:1/5 Step=140 Loss=0.00066 Rate=115.48 GlobalRate=58.25 Time=05:29:50
| Training Device=xla:1/7 Step=140 Loss=0.00066 Rate=115.34 GlobalRate=58.23 Time=05:29:50
| Training Device=xla:0/4 Step=140 Loss=0.00057 Rate=115.41 GlobalRate=58.24 Time=05:29:50
| Training Device=xla:1/1 Step=140 Loss=0.00066 Rate=115.44 GlobalRate=58.25 Time=05:29:50
| Training Device=xla:1/3 Step=140 Loss=0.00066 Rate=115.44 GlobalRate=58.25 Time=05:29:50
| Training Device=xla:0/4 Step=160 Loss=0.00047 Rate=7784.78 GlobalRate=66.46 Time=05:29:50
| Training Device=xla:0/2 Step=160 Loss=0.00047 Rate=7772.45 GlobalRate=66.46 Time=05:29:50
| Training Device=xla:1/1 Step=160 Loss=0.00055 Rate=7779.88 GlobalRate=66.47 Time=05:29:50
| Training Device=xla:0/6 Step=160 Loss=0.00047 Rate=7771.09 GlobalRate=66.44 Time=05:29:50
| Training Device=xla:1/3 Step=160 Loss=0.00055 Rate=7773.81 GlobalRate=66.47 Time=05:29:50
| Training Device=xla:1/7 Step=160 Loss=0.00055 Rate=7766.74 GlobalRate=66.44 Time=05:29:50
| Training Device=xla:0/0 Step=160 Loss=0.00047 Rate=7765.17 GlobalRate=66.47 Time=05:29:50
| Training Device=xla:1/5 Step=160 Loss=0.00055 Rate=7770.86 GlobalRate=66.47 Time=05:29:50
| Training Device=xla:1/1 Step=180 Loss=0.00047 Rate=11385.68 GlobalRate=74.68 Time=05:29:50
| Training Device=xla:1/5 Step=180 Loss=0.00047 Rate=11375.44 GlobalRate=74.68 Time=05:29:50
| Training Device=xla:0/6 Step=180 Loss=0.00041 Rate=11367.15 GlobalRate=74.65 Time=05:29:50
| Training Device=xla:0/2 Step=180 Loss=0.00041 Rate=11364.20 GlobalRate=74.68 Time=05:29:50
| Training Device=xla:1/3 Step=180 Loss=0.00047 Rate=11363.99 GlobalRate=74.68 Time=05:29:50
| Training Device=xla:1/7 Step=180 Loss=0.00047 Rate=11370.27 GlobalRate=74.65 Time=05:29:50
| Training Device=xla:0/0 Step=180 Loss=0.00041 Rate=11347.85 GlobalRate=74.69 Time=05:29:50
| Training Device=xla:0/4 Step=180 Loss=0.00041 Rate=11321.77 GlobalRate=74.67 Time=05:29:50
| Training Device=xla:1/7 Step=200 Loss=0.00041 Rate=12250.68 GlobalRate=82.85 Time=05:29:51
| Training Device=xla:1/3 Step=200 Loss=0.00041 Rate=12249.51 GlobalRate=82.88 Time=05:29:51
| Training Device=xla:0/4 Step=200 Loss=0.00035 Rate=12265.96 GlobalRate=82.87 Time=05:29:51
| Training Device=xla:1/1 Step=200 Loss=0.00041 Rate=12234.89 GlobalRate=82.88 Time=05:29:51
| Training Device=xla:0/2 Step=200 Loss=0.00035 Rate=12235.97 GlobalRate=82.87 Time=05:29:51
| Training Device=xla:0/0 Step=200 Loss=0.00035 Rate=12236.99 GlobalRate=82.89 Time=05:29:51
| Training Device=xla:1/5 Step=200 Loss=0.00041 Rate=12235.53 GlobalRate=82.88 Time=05:29:51
| Training Device=xla:0/6 Step=200 Loss=0.00035 Rate=12213.29 GlobalRate=82.85 Time=05:29:51
| Training Device=xla:1/1 Step=220 Loss=0.00036 Rate=12570.00 GlobalRate=91.06 Time=05:29:51
| Training Device=xla:0/2 Step=220 Loss=0.00031 Rate=12574.08 GlobalRate=91.06 Time=05:29:51
| Training Device=xla:0/6 Step=220 Loss=0.00031 Rate=12584.33 GlobalRate=91.03 Time=05:29:51
| Training Device=xla:0/4 Step=220 Loss=0.00031 Rate=12575.23 GlobalRate=91.05 Time=05:29:51
| Training Device=xla:0/0 Step=220 Loss=0.00031 Rate=12581.40 GlobalRate=91.07 Time=05:29:51
| Training Device=xla:1/3 Step=220 Loss=0.00036 Rate=12565.86 GlobalRate=91.06 Time=05:29:51
| Training Device=xla:1/5 Step=220 Loss=0.00036 Rate=12557.47 GlobalRate=91.07 Time=05:29:51
| Training Device=xla:1/7 Step=220 Loss=0.00036 Rate=12563.91 GlobalRate=91.03 Time=05:29:51
| Training Device=xla:1/3 Step=240 Loss=0.00032 Rate=13394.65 GlobalRate=99.25 Time=05:29:51
| Training Device=xla:0/6 Step=240 Loss=0.00027 Rate=13393.36 GlobalRate=99.21 Time=05:29:51
| Training Device=xla:0/4 Step=240 Loss=0.00027 Rate=13388.79 GlobalRate=99.23 Time=05:29:51
| Training Device=xla:1/1 Step=240 Loss=0.00032 Rate=13379.62 GlobalRate=99.25 Time=05:29:51
| Training Device=xla:1/7 Step=240 Loss=0.00032 Rate=13385.61 GlobalRate=99.21 Time=05:29:51
| Training Device=xla:1/5 Step=240 Loss=0.00032 Rate=13390.40 GlobalRate=99.25 Time=05:29:51
| Training Device=xla:0/0 Step=240 Loss=0.00027 Rate=13384.85 GlobalRate=99.26 Time=05:29:51
| Training Device=xla:0/2 Step=240 Loss=0.00027 Rate=13382.16 GlobalRate=99.24 Time=05:29:51
| Training Device=xla:1/5 Step=260 Loss=0.00029 Rate=14838.51 GlobalRate=107.43 Time=05:29:51
| Training Device=xla:0/0 Step=260 Loss=0.00025 Rate=14832.85 GlobalRate=107.44 Time=05:29:51
| Training Device=xla:0/2 Step=260 Loss=0.00025 Rate=14818.35 GlobalRate=107.42 Time=05:29:51
| Training Device=xla:1/7 Step=260 Loss=0.00029 Rate=14824.58 GlobalRate=107.39 Time=05:29:51
| Training Device=xla:0/4 Step=260 Loss=0.00025 Rate=14812.74 GlobalRate=107.41 Time=05:29:51
| Training Device=xla:1/1 Step=260 Loss=0.00029 Rate=14801.76 GlobalRate=107.43 Time=05:29:51
| Training Device=xla:1/3 Step=260 Loss=0.00029 Rate=14803.80 GlobalRate=107.43 Time=05:29:51
| Training Device=xla:0/6 Step=260 Loss=0.00025 Rate=14809.33 GlobalRate=107.39 Time=05:29:51
| Training Device=xla:0/0 Step=280 Loss=0.00023 Rate=15586.70 GlobalRate=115.61 Time=05:29:51
| Training Device=xla:1/5 Step=280 Loss=0.00026 Rate=15580.76 GlobalRate=115.61 Time=05:29:51
| Training Device=xla:0/6 Step=280 Loss=0.00023 Rate=15590.97 GlobalRate=115.56 Time=05:29:51
| Training Device=xla:1/3 Step=280 Loss=0.00026 Rate=15588.63 GlobalRate=115.60 Time=05:29:51
| Training Device=xla:1/1 Step=280 Loss=0.00026 Rate=15585.39 GlobalRate=115.60 Time=05:29:51
| Training Device=xla:1/7 Step=280 Loss=0.00026 Rate=15577.58 GlobalRate=115.56 Time=05:29:51
| Training Device=xla:0/2 Step=280 Loss=0.00023 Rate=15574.46 GlobalRate=115.60 Time=05:29:51
| Training Device=xla:0/4 Step=280 Loss=0.00023 Rate=15578.94 GlobalRate=115.59 Time=05:29:51
| Training Device=xla:1/5 Step=300 Loss=0.00024 Rate=15923.45 GlobalRate=123.77 Time=05:29:52
| Training Device=xla:0/0 Step=300 Loss=0.00020 Rate=15921.66 GlobalRate=123.78 Time=05:29:52
| Training Device=xla:1/3 Step=300 Loss=0.00024 Rate=15924.16 GlobalRate=123.76 Time=05:29:52
| Training Device=xla:0/6 Step=300 Loss=0.00020 Rate=15924.62 GlobalRate=123.72 Time=05:29:52
| Training Device=xla:1/1 Step=300 Loss=0.00024 Rate=15921.30 GlobalRate=123.76 Time=05:29:52
| Training Device=xla:0/2 Step=300 Loss=0.00020 Rate=15919.40 GlobalRate=123.76 Time=05:29:52
| Training Device=xla:0/4 Step=300 Loss=0.00020 Rate=15922.55 GlobalRate=123.75 Time=05:29:52
| Training Device=xla:1/7 Step=300 Loss=0.00024 Rate=15920.24 GlobalRate=123.72 Time=05:29:52
| Training Device=xla:1/5 Step=320 Loss=0.00022 Rate=14085.05 GlobalRate=131.91 Time=05:29:52
| Training Device=xla:1/1 Step=320 Loss=0.00022 Rate=14089.31 GlobalRate=131.90 Time=05:29:52
| Training Device=xla:0/6 Step=320 Loss=0.00019 Rate=14081.09 GlobalRate=131.85 Time=05:29:52
| Training Device=xla:0/2 Step=320 Loss=0.00019 Rate=14083.47 GlobalRate=131.90 Time=05:29:52
| Training Device=xla:0/0 Step=320 Loss=0.00019 Rate=14078.02 GlobalRate=131.92 Time=05:29:52
| Training Device=xla:1/3 Step=320 Loss=0.00022 Rate=14066.84 GlobalRate=131.90 Time=05:29:52
| Training Device=xla:0/4 Step=320 Loss=0.00019 Rate=14083.72 GlobalRate=131.89 Time=05:29:52
| Training Device=xla:1/7 Step=320 Loss=0.00022 Rate=14068.95 GlobalRate=131.86 Time=05:29:52
| Training Device=xla:0/0 Step=340 Loss=0.00018 Rate=15360.69 GlobalRate=140.07 Time=05:29:52
| Training Device=xla:1/5 Step=340 Loss=0.00020 Rate=15350.17 GlobalRate=140.06 Time=05:29:52
| Training Device=xla:0/2 Step=340 Loss=0.00018 Rate=15345.75 GlobalRate=140.05 Time=05:29:52
| Training Device=xla:0/6 Step=340 Loss=0.00018 Rate=15342.40 GlobalRate=140.00 Time=05:29:52
| Training Device=xla:0/4 Step=340 Loss=0.00018 Rate=15349.07 GlobalRate=140.03 Time=05:29:52
| Training Device=xla:1/1 Step=340 Loss=0.00020 Rate=15328.08 GlobalRate=140.05 Time=05:29:52
| Training Device=xla:1/7 Step=340 Loss=0.00020 Rate=15344.76 GlobalRate=140.00 Time=05:29:52
| Training Device=xla:1/3 Step=340 Loss=0.00020 Rate=15349.47 GlobalRate=140.05 Time=05:29:52
| Training Device=xla:0/4 Step=360 Loss=0.00016 Rate=13738.94 GlobalRate=148.15 Time=05:29:52
| Training Device=xla:0/2 Step=360 Loss=0.00016 Rate=13733.37 GlobalRate=148.16 Time=05:29:52
| Training Device=xla:0/0 Step=360 Loss=0.00016 Rate=13727.75 GlobalRate=148.19 Time=05:29:52
| Training Device=xla:1/7 Step=360 Loss=0.00019 Rate=13741.86 GlobalRate=148.12 Time=05:29:52
| Training Device=xla:1/5 Step=360 Loss=0.00019 Rate=13720.14 GlobalRate=148.18 Time=05:29:52
| Training Device=xla:1/1 Step=360 Loss=0.00019 Rate=13719.73 GlobalRate=148.17 Time=05:29:52
| Training Device=xla:1/3 Step=360 Loss=0.00019 Rate=13734.21 GlobalRate=148.17 Time=05:29:52
| Training Device=xla:0/6 Step=360 Loss=0.00016 Rate=13723.43 GlobalRate=148.11 Time=05:29:52
| Training Device=xla:1/3 Step=380 Loss=0.00018 Rate=14557.87 GlobalRate=156.29 Time=05:29:52
| Training Device=xla:0/4 Step=380 Loss=0.00015 Rate=14537.34 GlobalRate=156.27 Time=05:29:52
| Training Device=xla:1/7 Step=380 Loss=0.00018 Rate=14550.94 GlobalRate=156.24 Time=05:29:52
| Training Device=xla:1/1 Step=380 Loss=0.00018 Rate=14554.19 GlobalRate=156.29 Time=05:29:52
| Training Device=xla:0/2 Step=380 Loss=0.00015 Rate=14538.57 GlobalRate=156.29 Time=05:29:52
| Training Device=xla:0/6 Step=380 Loss=0.00015 Rate=14543.73 GlobalRate=156.23 Time=05:29:52
| Training Device=xla:0/0 Step=380 Loss=0.00015 Rate=14531.42 GlobalRate=156.31 Time=05:29:52
| Training Device=xla:1/5 Step=380 Loss=0.00018 Rate=14538.02 GlobalRate=156.30 Time=05:29:52
| Training Device=xla:1/5 Step=400 Loss=0.00016 Rate=15446.78 GlobalRate=164.42 Time=05:29:52
| Training Device=xla:0/2 Step=400 Loss=0.00014 Rate=15435.56 GlobalRate=164.41 Time=05:29:52
| Training Device=xla:1/7 Step=400 Loss=0.00016 Rate=15424.32 GlobalRate=164.35 Time=05:29:52
| Training Device=xla:0/4 Step=400 Loss=0.00014 Rate=15426.33 GlobalRate=164.39 Time=05:29:52
| Training Device=xla:1/1 Step=400 Loss=0.00016 Rate=15422.80 GlobalRate=164.41 Time=05:29:52
| Training Device=xla:0/6 Step=400 Loss=0.00014 Rate=15414.40 GlobalRate=164.35 Time=05:29:52
| Training Device=xla:1/3 Step=400 Loss=0.00016 Rate=15421.48 GlobalRate=164.41 Time=05:29:52
| Training Device=xla:0/0 Step=400 Loss=0.00014 Rate=15414.01 GlobalRate=164.43 Time=05:29:52
| Training Device=xla:1/5 Step=420 Loss=0.00016 Rate=15768.89 GlobalRate=172.53 Time=05:29:53
| Training Device=xla:1/1 Step=420 Loss=0.00016 Rate=15780.14 GlobalRate=172.52 Time=05:29:53
| Training Device=xla:1/3 Step=420 Loss=0.00016 Rate=15770.95 GlobalRate=172.52 Time=05:29:53
| Training Device=xla:0/4 Step=420 Loss=0.00013 Rate=15760.17 GlobalRate=172.50 Time=05:29:53
| Training Device=xla:1/7 Step=420 Loss=0.00016 Rate=15755.31 GlobalRate=172.46 Time=05:29:53
| Training Device=xla:0/2 Step=420 Loss=0.00013 Rate=15746.12 GlobalRate=172.52 Time=05:29:53
| Training Device=xla:0/6 Step=420 Loss=0.00013 Rate=15759.09 GlobalRate=172.46 Time=05:29:53
| Training Device=xla:0/0 Step=420 Loss=0.00013 Rate=15734.96 GlobalRate=172.54 Time=05:29:53
| Training Device=xla:0/0 Step=440 Loss=0.00013 Rate=15441.93 GlobalRate=180.64 Time=05:29:53
| Training Device=xla:0/6 Step=440 Loss=0.00013 Rate=15426.54 GlobalRate=180.55 Time=05:29:53
| Training Device=xla:1/5 Step=440 Loss=0.00015 Rate=15400.35 GlobalRate=180.63 Time=05:29:53
| Training Device=xla:0/2 Step=440 Loss=0.00013 Rate=15414.67 GlobalRate=180.62 Time=05:29:53
| Training Device=xla:1/7 Step=440 Loss=0.00015 Rate=15400.81 GlobalRate=180.56 Time=05:29:53
| Training Device=xla:1/1 Step=440 Loss=0.00015 Rate=15391.49 GlobalRate=180.62 Time=05:29:53
| Training Device=xla:0/4 Step=440 Loss=0.00013 Rate=15397.13 GlobalRate=180.60 Time=05:29:53
| Training Device=xla:1/3 Step=440 Loss=0.00015 Rate=15397.36 GlobalRate=180.62 Time=05:29:53
| Training Device=xla:1/1 Step=460 Loss=0.00014 Rate=14698.30 GlobalRate=188.70 Time=05:29:53
| Training Device=xla:0/2 Step=460 Loss=0.00012 Rate=14692.66 GlobalRate=188.70 Time=05:29:53
| Training Device=xla:0/4 Step=460 Loss=0.00012 Rate=14687.41 GlobalRate=188.68 Time=05:29:53
| Training Device=xla:1/7 Step=460 Loss=0.00014 Rate=14686.04 GlobalRate=188.64 Time=05:29:53
| Training Device=xla:0/0 Step=460 Loss=0.00012 Rate=14691.31 GlobalRate=188.73 Time=05:29:53
| Training Device=xla:1/3 Step=460 Loss=0.00014 Rate=14672.47 GlobalRate=188.70 Time=05:29:53
| Training Device=xla:1/5 Step=460 Loss=0.00014 Rate=14673.90 GlobalRate=188.71 Time=05:29:53
| Training Device=xla:0/6 Step=460 Loss=0.00012 Rate=14676.58 GlobalRate=188.63 Time=05:29:53
| Training Device=xla:1/7 Step=480 Loss=0.00013 Rate=15559.37 GlobalRate=196.72 Time=05:29:53
| Training Device=xla:0/0 Step=480 Loss=0.00011 Rate=15552.65 GlobalRate=196.81 Time=05:29:53
| Training Device=xla:1/3 Step=480 Loss=0.00013 Rate=15562.99 GlobalRate=196.79 Time=05:29:53
| Training Device=xla:0/4 Step=480 Loss=0.00011 Rate=15535.84 GlobalRate=196.77 Time=05:29:53
| Training Device=xla:0/6 Step=480 Loss=0.00011 Rate=15554.44 GlobalRate=196.72 Time=05:29:53
| Training Device=xla:1/1 Step=480 Loss=0.00013 Rate=15535.16 GlobalRate=196.79 Time=05:29:53
| Training Device=xla:0/2 Step=480 Loss=0.00011 Rate=15533.42 GlobalRate=196.78 Time=05:29:53
| Training Device=xla:1/5 Step=480 Loss=0.00013 Rate=15530.35 GlobalRate=196.80 Time=05:29:53
| Training Device=xla:1/5 Step=500 Loss=0.00012 Rate=16054.18 GlobalRate=204.88 Time=05:29:53
| Training Device=xla:0/0 Step=500 Loss=0.00011 Rate=16038.05 GlobalRate=204.90 Time=05:29:53
| Training Device=xla:0/6 Step=500 Loss=0.00011 Rate=16046.64 GlobalRate=204.80 Time=05:29:53
| Training Device=xla:1/3 Step=500 Loss=0.00012 Rate=16040.52 GlobalRate=204.87 Time=05:29:53
| Training Device=xla:0/4 Step=500 Loss=0.00011 Rate=16037.26 GlobalRate=204.85 Time=05:29:53
| Training Device=xla:1/1 Step=500 Loss=0.00012 Rate=16032.57 GlobalRate=204.87 Time=05:29:53
| Training Device=xla:0/2 Step=500 Loss=0.00011 Rate=16029.19 GlobalRate=204.86 Time=05:29:53
| Training Device=xla:1/7 Step=500 Loss=0.00012 Rate=16029.97 GlobalRate=204.80 Time=05:29:53
| Training Device=xla:1/5 Step=520 Loss=0.00012 Rate=14890.85 GlobalRate=212.94 Time=05:29:53
| Training Device=xla:0/0 Step=520 Loss=0.00010 Rate=14879.15 GlobalRate=212.95 Time=05:29:53
| Training Device=xla:1/7 Step=520 Loss=0.00012 Rate=14882.62 GlobalRate=212.85 Time=05:29:53
| Training Device=xla:0/2 Step=520 Loss=0.00010 Rate=14880.55 GlobalRate=212.92 Time=05:29:53
| Training Device=xla:0/4 Step=520 Loss=0.00010 Rate=14882.91 GlobalRate=212.90 Time=05:29:53
| Training Device=xla:0/6 Step=520 Loss=0.00010 Rate=14875.27 GlobalRate=212.85 Time=05:29:53
| Training Device=xla:1/1 Step=520 Loss=0.00012 Rate=14867.22 GlobalRate=212.92 Time=05:29:53
| Training Device=xla:1/3 Step=520 Loss=0.00012 Rate=14870.77 GlobalRate=212.93 Time=05:29:53
| Training Device=xla:0/0 Step=540 Loss=0.00010 Rate=15079.00 GlobalRate=221.01 Time=05:29:54
| Training Device=xla:1/3 Step=540 Loss=0.00011 Rate=15082.76 GlobalRate=220.98 Time=05:29:54
| Training Device=xla:0/4 Step=540 Loss=0.00010 Rate=15068.38 GlobalRate=220.96 Time=05:29:54
| Training Device=xla:0/6 Step=540 Loss=0.00010 Rate=15070.08 GlobalRate=220.90 Time=05:29:54
| Training Device=xla:1/1 Step=540 Loss=0.00011 Rate=15074.44 GlobalRate=220.98 Time=05:29:54
| Training Device=xla:0/2 Step=540 Loss=0.00010 Rate=15069.47 GlobalRate=220.97 Time=05:29:54
| Training Device=xla:1/7 Step=540 Loss=0.00011 Rate=15056.05 GlobalRate=220.90 Time=05:29:54
| Training Device=xla:1/5 Step=540 Loss=0.00011 Rate=15054.51 GlobalRate=220.99 Time=05:29:54
| Training Device=xla:0/2 Step=560 Loss=0.00009 Rate=15650.30 GlobalRate=229.03 Time=05:29:54
| Training Device=xla:0/0 Step=560 Loss=0.00009 Rate=15630.48 GlobalRate=229.06 Time=05:29:54
| Training Device=xla:1/5 Step=560 Loss=0.00011 Rate=15644.80 GlobalRate=229.05 Time=05:29:54
| Training Device=xla:0/6 Step=560 Loss=0.00009 Rate=15640.85 GlobalRate=228.95 Time=05:29:54
| Training Device=xla:1/3 Step=560 Loss=0.00011 Rate=15628.78 GlobalRate=229.03 Time=05:29:54
| Training Device=xla:1/1 Step=560 Loss=0.00011 Rate=15640.70 GlobalRate=229.03 Time=05:29:54
| Training Device=xla:0/4 Step=560 Loss=0.00009 Rate=15618.58 GlobalRate=229.01 Time=05:29:54
| Training Device=xla:1/7 Step=560 Loss=0.00011 Rate=15621.50 GlobalRate=228.95 Time=05:29:54
| Training Device=xla:1/7 Step=580 Loss=0.00011 Rate=15602.28 GlobalRate=236.99 Time=05:29:54
| Training Device=xla:0/4 Step=580 Loss=0.00009 Rate=15596.55 GlobalRate=237.05 Time=05:29:54
| Training Device=xla:1/1 Step=580 Loss=0.00011 Rate=15587.85 GlobalRate=237.07 Time=05:29:54
| Training Device=xla:0/2 Step=580 Loss=0.00009 Rate=15570.52 GlobalRate=237.07 Time=05:29:54
| Training Device=xla:0/6 Step=580 Loss=0.00009 Rate=15578.39 GlobalRate=236.99 Time=05:29:54
| Training Device=xla:0/0 Step=580 Loss=0.00009 Rate=15565.67 GlobalRate=237.10 Time=05:29:54
| Training Device=xla:1/5 Step=580 Loss=0.00011 Rate=15576.06 GlobalRate=237.09 Time=05:29:54
| Training Device=xla:1/3 Step=580 Loss=0.00011 Rate=15555.55 GlobalRate=237.07 Time=05:29:54
Epoch 1 train end 05:29:54
Epoch 1 test end 05:29:56, Accuracy=100.00
Max Accuracy: 100.00%
```

**Comment by ManfeiBai** [comment]
Hi, Jiewen, @alanwaketan  do you think its a good idea to change `` to ``

what we want to do in this PR is to delete all the tensorflow code for profile, so that we could delete tensorflow code in the next step

since we start to use `tensorboard_plugin_profile`, please correct me or update PR if you think we have any other tensorflow code could be deleted too

**Comment by alanwaketan** [comment]
> Hi, Jiewen, @alanwaketan do you think its a good idea to change `to`
> 
> what we want to do in this PR is to delete all the tensorflow code for profile, so that we could delete tensorflow code in the next step
> 
> since we start to use `tensorboard_plugin_profile`, please correct me or update PR if you think we have any other tensorflow code could be deleted too

Can you show profiler outputs w.r.t to the trace view before and after this change? You can grep the cheat sheet on how to do profiling.

File: torch_xla/csrc/init_python_bindings.cpp [hunk 1]
```diff
@@ -855,7 +855,7 @@ void BuildProfilerSubmodule(py::module* m) {
         {
           NoGilSection nogil;
           for (int i = 0; i <= timeout_s / interval_s; i++) {
-            status = tensorflow::profiler::pywrap::Trace(
+            status = tsl::profiler::CaptureRemoteTrace(
```
Comment (miladm) [inline torch_xla/csrc/init_python_bindings.cpp:858]: @cliveverghese can you point us to the right openxla API to use in place of this line please?

**Comment by ManfeiBai** [comment]
test results: All checks have failed
- ci/circleci: linter_check : https://circleci.com/gh/pytorch/xla/43882
- ci/circleci: pytorch_xla_run_build : https://circleci.com/gh/pytorch/xla/43881

**Comment by ManfeiBai** [comment]
this PR is too old and this modification could not enable profile like the old style, we still need the full function Trace, so will close this PR and reopen once needed

**Pull request closed without merging**